src/docs/content/extras/canary.md

```diff
@@ -0,0 +1,246 @@
+
+## Pre-Requisites:
+
+Deploy [Mushop]({{< ref "quickstart/kubernetes.md" >}})
+
+Download and install [Istio Service Mesh]({{< ref "istio.md" >}})
+
+## Deploy storefront beta
+ 
+ - Create a deployment with the below config and name it mushop-storefrontv2.
+   Note: We will use the same labels of storefront original.
+
+  ```shell--macos-linux
+    cat << EOF | kubectl apply -f -
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      labels:
+        app.kubernetes.io/instance: mushop
+        app.kubernetes.io/name: storefront
+      name: mushop-storefrontv2
+      namespace: mushop
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: storefront
+          app.kubernetes.io/instance: mushop
+          app.kubernetes.io/name: storefront
+      template:
+        metadata:
+          labels:
+            app: storefront
+            app.kubernetes.io/instance: mushop
+            app.kubernetes.io/name: storefront
+            version: 2.1.3
+        spec:
+          containers:
+          - image: syd.ocir.io/ociateam/mushop/storefront:beta
+            imagePullPolicy: Always
+            name: storefront
+    EOF
+  ```
+
+
+## Create Istio resources
+
+- Deploy a Gateway resource
+
+  ```shell--macos-linux
+    cat << EOF | kubectl apply -f -
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: gateway
+      namespace: mushop
+    spec:
+      selector:
+        istio: ingressgateway
+      servers:
+        - port:
+            number: 80
+            name: http
+            protocol: HTTP
+          hosts:
+            - '*'
+    EOF
+    ```
+
+- Deploy a VirtualService and DestinationRules
+  
+  ```shell--macos-linux
+    cat <<EOF | kubectl apply -f -
+    apiVersion: networking.istio.io/v1alpha3
+    kind: VirtualService
+    metadata:
+      name: edge
+      namespace: mushop
+    spec:
+      hosts:
+        - '*'
+      gateways:
+        - gateway
+      http:
+      - match:
+        - uri:
+            prefix: /api
+        route:
+        - destination:
+            host: mushop-api.mushop.svc.cluster.local
+      - match:
+        - uri:
+            prefix: /assets
+        rewrite:
+          uri: /
+        route:
+        - destination:
+            host: mushop-assets.mushop.svc.cluster.local
+      - route:
+        - destination:
+            host: mushop-storefront.mushop.svc.cluster.local
+            port:
+              number: 80
+            subset: original
+          weight: 50
+        - destination:
+            host: mushop-storefront.mushop.svc.cluster.local
+            port:
```

Comment:
Is `port` necessary here? It appears to be omitted from the other routes

Reply:
tested without port and works fine. will remove the port field.